### PR TITLE
fix: Enable login timeout

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

This will also adjust the timeout in web.xml in Tomcat according to [rhn_web.conf](https://github.com/uyuni-project/uyuni/blob/d59de49972ee98d8542c852e4e9d24f4d8f5bd42/web/conf/rhn_web.conf#L29-L31).

According to the [Jinja docs](https://jinja.palletsprojects.com/en/3.0.x/templates/#math) we can use `//` to return a truncated integer result of the division that is made.
> //
Divide two numbers and return the truncated integer result. {{ 20 // 7 }} is 2.

### CODE REVIEW

File: modules/server/main.tf [hunk 1]
```diff
@@ -78,6 +78,7 @@ module "server" {
     server_registration_code       = var.server_registration_code
     accept_all_ssl_protocols       = var.accept_all_ssl_protocols
     login_timeout                  = var.login_timeout
+    login_timeout_minutes          = var.login_timeout / 60
```
Comment (nodeg) [inline modules/server/main.tf:81]: I am not sure if this is possible the way I did it.

Comment (srbarrios) [reply modules/server/main.tf:81]: For that kind of calculation, just do it inside the sls. Using Jinja2.

**Comment by nodeg** [comment]
I am not sure if I should touch `/etc/tomcat/web.xml`, too?

File: salt/server/tomcat.sls [hunk 2]
```diff
@@ -17,6 +17,17 @@ tomcat_config:
 
 {% endif %}
 
+{% if grains.get('login_timeout') %}
+extend_login_timeout:
+  file.replace:
+    - name: /srv/tomcat/webapps/rhn/WEB-INF/web.xml
+    - pattern: <session-timeout>*
+    - repl: <session-timeout>{{ grains['login_timeout_minutes'] }}</session-timeout>
```
Comment (srbarrios) [inline salt/server/tomcat.sls:25]:
```suggestion
    - repl: <session-timeout>{{ grains['login_timeout'] / 60 }}</session-timeout>
```

File: modules/server/main.tf (hunk 1 above)
Comment (nodeg) [reply modules/server/main.tf:81]: Ok, will do.

Comment (nodeg) [reply modules/server/main.tf]: Fixed.

**Review by srbarrios — APPROVED** [review]

**Pull request merged**